src/CardContext.js

```diff
@@ -40,11 +40,13 @@ const CardContextProvider = (props) => {
     currentRound: 1, // seven rounds per phase
     currentPhase: 1, // four phases per game
     numPlayers: null,
-    playerOne: { current: [], complete: [], score },
-    playerTwo: { current: [], complete: [], score },
-    playerThree: { current: [], complete: [], score },
-    playerFour: { current: [], complete: [], score },
+    playerOne: { current: [], complete: [], score, coins: 0 },
+    playerTwo: { current: [], complete: [], score, coins: 0 },
+    playerThree: { current: [], complete: [], score, coins: 0 },
+    playerFour: { current: [], complete: [], score, coins: 0 },
     roundSubmits: [0, 0, 0, 0],
+    coins: [0, 0, 0, 0],
```

Comment:
What's the difference between state.player*.coins (L43-L46) and the array in state.coins (L48)?

Reply:
state.player*coins can be removed. Early attempt